rocketpy/AeroSurfaces.py

```diff
@@ -1901,7 +1867,7 @@ def aerodynamicInfo(self):
         print(f"\nAerodynamic Information of {self.name}")
         print("-------------------------------")
         print(f"Tail Center of Pressure Position in Local Coordinates: {self.cp} m")
-        print(f"Tail Lift Coefficient Slope: {self.clalpha:.3f} 1/rad")
+        print(f"Tail Lift Coefficient Slope: {self.clalpha(0):.3f} 1/rad")
```

Comment:
Shall we specify that the coefficient slope is being taken in alpha==0 ? (I understand that the alpha is the angle of attack)

Reply:
`clalpha` is a function of mach, so `clalpha(0)` is the Tail Lift Coefficient Slope at Mach 0 (or at speed 0).
I think it is a good idea to specify that it is at mach 0

Reply:
Ok, sorry for my mistake and thanks for solving this one in the last commit :)